site/src/routes/_docs.svtext

```diff
@@ -115,6 +115,22 @@ const compiled = svelte.compile(
 
 > If you don't like the `.svx` file extension, fear not, it is easily customised.
 
+### `mdsvex with sapper`
+
+To use mdsvex with sapper you need to put the above configurations both on the client and server of the rollup/webpack configurations, and you need to put the `--ext '.svelte .svx'` in the sapper scripts of package.json
+
+Or you can use the templates:
+- [Rollup](https://github.com/pngwn/sapper-mdsvex-template)
+
+  ```bash
+  npx degit "pngwn/sapper-mdsvex-template" my-app
+  ```
+- [Webpack](https://github.com/shiryel/sapper-mdsvex-template-webpack)
+  
+  ```bash
+  npx degit "shiryel/sapper-mdsvex-template-webpack" my-app
+  ```
+
```

Comment:
There is a placeholder `Integrations` section on line 629. Could you move this section down there under a `### sapper` heading?